topcoffea/modules/corrections.py

```diff
@@ -75,7 +75,7 @@
 SFevaluator = extLepSF.make_evaluator()
 
 
-def GetLeptonSF(pt1, eta1, type1, pt2, eta2, type2, pt3=None, eta3=None, type3=None, year=2018, sys=0):
+def GetLeptonSF(pt1, eta1, type1, pt2, eta2, type2, pt3=None, eta3=None, type3=None, pt4=None, eta4=None, type4=None, year=2018, sys=0):
```

Comment:
What are `type3` and `type4`? It looks like below that `type4` is the lepton flavor. Should we pick better names?

Reply:
Yes, type=flavor, I have updated the name